docs/experimental-machine-learning/host-risk-score.md

```diff
@@ -1,105 +1,192 @@
+- Confirm the transform is working as expected by navigating to `Management / Dev Tools` and ensuring the target index exists. You should see documents starting to appear in the index if there is ongoing alerting activity associated with hosts.
+
+<div style="margin-left: 40px">   
+<i>sample test query</i>
+   <pre style="margin-top:-2px"><code>GET ml_host_risk_score_latest_&lt;your-space-name&gt;/_search
+</code></pre></div>
 
-* Navigate to `Transforms` under `Management` -> `Stack Management`. For the transform with the ID `ml_hostriskscore_latest_transform_<your-space-name>`, under `Actions`, click `Start`. 
-* Verify that the Transform started as expected by ensuring that documents are appearing in the destination index of the Transform, eg: using the Search/Count APIs:
+<h3 id="import-dashboards">7. Import dashboards</h3>
 
+- Navigate to `Management / Stack Management / Kibana / Saved Objects` in Kibana.
+- Click on `Import` and import the `ml_hostriskscore_dashboards.ndjson` file.
+- Navigate to `Analytics / Dashboard`.
+- Confirm you can see a dashboard named `Current Risk Scores for Hosts`, which displays the current list (Top 20) of suspicious hosts in your environment.
+- Confirm you can see a dashboard named `Drilldown of Host Risk Score`, which allows you to further drill down into details of the risk associated with a particular host of interest.
+
+<h3 id="enable-kibana">8. Enable Kibana features</h3>
+
+There are two features in Kibana in relation to Host Risk Scoring.
+
+_Host Risk Score card on the Overview page_
+![Host Risk Score card](./images/0a_host_risk_score_card.png)
+
+_Host Risk Keyword on Alert Details Flyout_
+![Host Risk Keyword](./images/0b_alert_summary.png)
+
+To enable the _Host Risk Score card on the Overview page_, and the _Host Risk Keyword on Alert Details Flyout_, you will need to add the following configuration to `kibana.yml`.
 
 ```
-GET ml_host_risk_score_latest_<your-space-name>/_search (or _count)
+xpack.securitySolution.enableExperimental: ['riskyHostsEnabled']
 ```
 
-#### 9. Import the dashboards
+#### Instructions to modify `kibana.yml` on Elastic Cloud
+
+1. Navigate to your deployment on the cloud
+![Navigate to deployment](./images/1_create_deployment.png)
+
+
+2. Click on Kibana on the sidebar and click on Edit configuration on your Kibana instance
+![Edit Kibana config](./images/2_edit_configuration.png)
+
+
+3. Click on Edit user settings
+![Edit user settings](./images/3_edit_user_settings.png)
+
+
+4. Modify Kibana configuration by adding `xpack.securitySolution.enableExperimental: ['riskyHostsEnabled']`
+![Modify Kibana configuration](./images/4_add_flag.png)
+
+
+5. Save updated Kibana settings
+![Save updated Kibana settings](./images/5_save_settings.png)
+   
+
+6. Confirm activity finished
```

Comment:
Can we please highlight the activity finished step in this screenshot?

Reply:
updated image